core/templates/pages/about-page/about-page.constants.ts

```diff
@@ -256,6 +267,7 @@ export const AboutPageConstants = {
     'Namuli Joyce',
     'Naveen Kumar Shukla',
     'Netaji Kancharapu',
+    'Nicolas Skirkey',
```

Comment:
There are a few people here who weren't in the AUTHORS/CONTRIBUTORS change--why the discrepancy?

Reply:
This name is on both the AUTHORS and CONTRIBUTORS file.  It might have been a missing entry from the previous releases and the script automatically added it I think.

Reply:
Thanks for checking.